pkg/cache/v3/delta.go

```diff
@@ -53,29 +53,26 @@ func createDeltaResponse(ctx context.Context, req *DeltaRequest, state stream.St
 		}
 
 		// Compute resources for removal
+		// The resource version can be set to "" here to trigger a removal even if never returned before
```

Comment:
Is this true? Does a removal need to happen if we've never sent the resource before? The protocol spec is a bit vague around this so want to make sure we make a good decision here.

Reply:
If the resource was never sent the version will not be set here 
The only case where it will happen now is when we want to explicitly trigger this "deletion", as done at https://github.com/envoyproxy/go-control-plane/pull/559/files#diff-5119f0079a6d464b74323939220b02d9a7aebcafd906e2e5be0161637afe5022R243